examples/async_transcribe_local_media_file/async_transcribe_local_media_file.js

```diff
@@ -0,0 +1,62 @@
+const revai = require('revai-node-sdk');
+const fs = require('fs');
+const token = require('../config/config.json').access_token;
+
+(async () => {  
+    /* Initialize your client with your revai access token */
+    var client = new revai.RevAiApiClient(token);
+
+    /* Get account details */
+    var account = await client.getAccount();
+    console.log(`Account: ${account.email}`);
+    console.log(`Balance: ${account.balance_seconds} seconds`);
+
+    const jobOptions = {
+        metadata: "InternalOrderNumber=123456789",
+        callback_url: "https://jsonplaceholder.typicode.com/posts",
+        skip_diarization: true,
+        custom_vocabularies: [{
+            phrases: [
+                "add",
+                "custom",
+                "vocabularies",
+                "here"
+                ]
+            }]
+    };
+
+    /* Media may be submitted from a local file */
+    var job = await client.submitJobLocalFile("../resources/example.mp3", jobOptions);
+
+    console.log(`Job Id: ${job.id}`);
+    console.log(`Status: ${job.status}`);
+    console.log(`Created On: ${job.created_on}`);
+
+    /* Waits 5 seconds between each status check to see if job is complete.
```

Comment:
are comments supposed to start on this line?

or are they supposed to be like 
```
/*
* Media may be..
*/
```
I'm just as unfamiliar with typescript as you, so question for @kbridbur or @amikofalvy

Reply:
They should go on the next line. Single line comments begin with `//`